src/utils/url.ts

```diff
@@ -208,13 +213,15 @@ const getUpdatedHashParams = (updates: UrlHashParamUpdatesType) => {
     const newHashParams = new URLSearchParams(currentHashParams as Record<string, string>);
 
     for (const [key, value] of Object.entries(updates)) {
-        if (isEmptyOrFalsy(value)) {
+        if (value === URL_HASH_PARAMS_DEFAULT[key as HASH_PARAM_NAMES]) {
             newHashParams.delete(key);
         } else {
             newHashParams.set(key, String(value));
         }
     }
 
+    console.log(newHashParams.toString());
+
```

Comment:
we should remove this debug print (if it was added only for debugging changes in this PR?)
```suggestion
```

Reply:
shall we address this?